jina/peapods/runtimes/asyncio/rest/models.py

```diff
@@ -31,9 +31,11 @@ def build_model_from_pb(name, pb_model):
 
 
 class JinaRequestModel(BaseModel):
-    data: Union[List[JinaDocumentModel], List[Dict[str, Any]], List[str], List[bytes]]
-    request_size: Optional[int] = 0
-    mime_type: Optional[str] = None
+    # To avoid an error while loading the request model schema on swagger, we've added an example.
+    data: Union[List[JinaDocumentModel], List[Dict[str, Any]], List[str], List[bytes]] = \
+        Field(..., example=[Document().dict()])
+    request_size: Optional[int] = 1
```

Comment:
this should not be `1`, otherwise the REST interface will be unbelievable slow.

use set_client_parser().parser([]).request_size to get default request size